tests/src/main/java/fixtures/bodycomplex/models/GoblinSharkColor.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Static value RED for GoblinSharkColor.
      */
-    //public static final GoblinSharkColor RED = fromString("RED");
+//    public static final GoblinSharkColor RED = fromString("RED");
```

Comment:
Both `RED` and `red` enums are now commented due to merging order of my [changes](https://github.com/Azure/autorest.java/commit/40cefe2675927b3191f13c131d5aa3b5a1af3156#diff-49b1b1ae6d90a899f8a4f6940b106feeL34) and @weidongxu-microsoft 's [changes](https://github.com/Azure/autorest.java/commit/0800fc5010ef1c4f03ce103864575790a00651da#diff-49b1b1ae6d90a899f8a4f6940b106feeL29). One of the enums should be uncommented.